modules/web-console-cso.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * web_console/scan-from-web-console.adoc
+
+[id="web-console-cso_{context}"]
+= Run Container Security Operator in {product-title}
```

Comment:
I'd consider whether saying "in {product-title}" can be dropped, because the whole doc set is for OCP.

Also, modules should start with gerunds, so I'd go with "Running the Container Security Operator"

Reply:
Yes.

Reply:
Both done.